src/assets/scripts/client/aircraft/AircraftModel.js

```diff
@@ -1047,9 +1045,9 @@ export default class AircraftModel {
             head: 0
         };
 
-        const { wind } = AirportController.airport_get();
-        // const wind = airport.wind;
-        const angle = this.fms.currentRunway.calculateCrosswindAngleForRunway(wind.angle);
+        const { wind } = this.fms.arrivalAirportModel || this.fms.departureAirportModel;
+        const runwayModel = this.fms.arrivalRunwayModel || this.fms.departureRunwayModel;
```

Comment:
I feel like these two lines should be encapsulated in methods. Destructuring with `||` here looks icky and is a pain in the ass to test.

Reply:
I agree in principle. This particular method is one I have an issue with because it asks for properties of an airport and of a runway without specifying which airport or which runway they're querying. I think that is a bad practice and the method itself should be changed. I would not want to add a new method that does the `||` logic because frankly that's not an operation we should be doing. We should be requesting information more specifically than that. As a similar example, you'll see that the `Fms.currentRunway` getter is gone for the same reason. There are better and more organized ways to get what we need. I just didn't take the time to redo this wind method. I can add a todo if nothing else because it _is_ quite gross.

Reply:
seems I left a similar comment above just now.  we can discuss later...